docs/index.md

```diff
@@ -66,13 +66,14 @@ Discover the practical application of Powerloom through Pooler, a Uniswap V2 Das
 ## Building with Powerloom
 
 To begin using the Powerloom Protocol, you need to understand the workflow and architecture of the system. This involves familiarizing yourself with the different components and how they interact within the protocol.
-The best way to start is by exploring the existing implementations present in [existing-implementations](/docs/category/existing-implementations), understanding how they work, and how you can leverage them to build your own use cases. We have also provided examples of how you can build your own use cases in the [building-use-cases](/docs/category/building-a-new-use-case) section.
+The best way to start is by exploring the [existing-implementations](/docs/category/existing-implementations), understanding how they work, and how you can leverage them to [build your own use cases](/docs/category/building-a-new-use-case).
 
 ### Utilizing Existing Implementations to Build Your Data Application
 
-Although the data generated by the Powerloom Protocol is decentralized and accessible to anyone, it's not always easy to understand how to use it. To make it easier for developers, each snapshotter node comes with a set of APIs that can be used to access the data. These APIs are designed to be easy to use and can be employed to build data applications such as dashboards, bots, aggregators, and insights trackers.
+The data generated by the Powerloom Protocol is decentralized and accessible to everyone. It’s tough to understand how to make best use of the data generated.
```

Comment:
https://github.com/PowerLoom/docs/pull/29

> It’s tough to understand how to make best use of the data generated.

Definitely not friendly language for any one landing up on the docs. Rephrase for a better one or remove this altogether.